tox.ini

```diff
@@ -1,8 +1,9 @@
 [tox]
-envlist = py39,py310
+envlist = py310,py311,py312
 isolated_build = True
 
 [gh]
 python =
-    3.9 = py39, type
```

Comment:
not sure what the type was for. Do I need to add this back?

Reply:
IMO delete it here and discuss in #43